### PR TITLE
Updated redis tests to fail when redis is not running

### PR DESCRIPTION
**Pull request by Noviny**
Previously they would just hang indefinitely

### CODE REVIEW

**Comment by Noviny** [comment]
Currently unsure if we should update `packages/session-store-redis/src/index.ts` line 12, which is

```
    async connect() {
      client.on('error', err => console.log('Redis Client Error', err)); // this one

      await client.connect();
    },
```

and sort of just swallows errors, and seems to lead to reattempting rather than failing out.

File: tests/api-tests/auth-stored-session.test.ts [hunk 1]
```diff
@@ -99,6 +100,19 @@ async function login(
 }
 
 describe('Auth testing', () => {
+  beforeAll(async () => {
+    try {
+      await fetch('http:127.0.0.1:6379', {
```
Comment (dcousens) [inline tests/api-tests/auth-stored-session.test.ts:105]:
```suggestion
      await fetch('http://127.0.0.1:6379', {
```

File: tests/api-tests/auth-stored-session.test.ts [hunk 2]
```diff
@@ -99,6 +100,19 @@ async function login(
...
 describe('Auth testing', () => {
+  beforeAll(async () => {
+    try {
+      await fetch('http://127.0.0.1:6379', {
+        method: 'post',
+        body: 'ping',
+      });
+    } catch (e) {
+      if ((e as Error).message.includes('ECONNREFUSED')) {
+        throw new Error('redis is not running on this computer so this test suite cannot be run');
```
Comment (dcousens) [inline tests/api-tests/auth-stored-session.test.ts:111]:
```suggestion
        throw new Error('redis not found: ECONNREFUSED');
```

Comment (dcousens) [reply tests/api-tests/auth-stored-session.test.ts]: if debugging a redis instance, this is usually an easier search term

**Review by dcousens — APPROVED** [review]

**Review by dcousens — APPROVED** [review]

**Pull request merged**